### ISSUE TITLE
Not working anymore

### ISSUE THREAD

**Issue by naoliv** [k5md/Steam-Bulk-Sell-webextension#13]
I am unsure if steam changed something, but the checkboxes are gone:

![](https://github.com/k5md/Steam-Bulk-Sell-webextension/assets/982338/b762a0b9-ebf0-481f-a0e2-455387f84945)

This is on Firefox 114

Thanks!

**Comment by naoliv** [comment]
Hi!

Right after loading the page I am seeing this:

![](https://github.com/k5md/Steam-Bulk-Sell-webextension/assets/982338/40a4d294-fbe6-4416-9186-0d02daeb7273)

Text:
```
Layout was forced before the page was fully loaded. If stylesheets are not yet loaded this may cause a flash of unstyled content. [jquery-1.11.1.min.js:4:29551](https://community.cloudflare.steamstatic.com/public/javascript/jquery-1.11.1.min.js?v=.isFTSRckeNhC&_cdn=cloudflare)
Google Analytics and Tag Manager is being shimmed by Firefox. See https://bugzilla.mozilla.org/show_bug.cgi?id=1713687 for details. [sandbox eval code:1:9](chrome://devtools/content/webconsole/sandbox%20eval%20code)
Cookie “timezoneOffset” does not have a proper “SameSite” attribute value. Soon, cookies without the “SameSite” attribute or with an invalid value will be treated as “Lax”. This means that the cookie will no longer be sent in third-party contexts. If your application depends on this cookie being available in such contexts, please add the “SameSite=None“ attribute to it. To know more about the “SameSite“ attribute, read https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie/SameSite [shared_global.js:3858:83](https://community.cloudflare.steamstatic.com/public/shared/javascript/shared_global.js?v=-Hg78zgW4Qwa&l=english&_cdn=cloudflare)
GET
wss://community.steam-api.com/websocket/

Firefox can’t establish a connection to the server at wss://community.steam-api.com/websocket/. [market.js:1901:14](https://community.cloudflare.steamstatic.com/public/javascript/market.js?v=PPAhWEfZ1Yy2&l=english&_cdn=cloudflare)
WebSocket error. Retry in 100891ms.
```

Then clicking on `Sell an item`, this appears:
![](https://github.com/k5md/Steam-Bulk-Sell-webextension/assets/982338/d731f84d-b9b8-4fcb-bbe4-1e77174c5828)


Text:
```
Content Security Policy: Ignoring ‘x-frame-options’ because of ‘frame-ancestors’ directive.
Some cookies are misusing the recommended “SameSite“ attribute 2
Cookie “timezoneOffset” does not have a proper “SameSite” attribute value. Soon, cookies without the “SameSite” attribute or with an invalid value will be treated as “Lax”. This means that the cookie will no longer be sent in third-party contexts. If your application depends on this cookie being available in such contexts, please add the “SameSite=None“ attribute to it. To know more about the “SameSite“ attribute, read https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie/SameSite [shared_global.js:3858:83](https://community.cloudflare.steamstatic.com/public/shared/javascript/shared_global.js?v=-Hg78zgW4Qwa&l=english&_cdn=cloudflare)
Cookie “strInventoryLastContext” does not have a proper “SameSite” attribute value. Soon, cookies without the “SameSite” attribute or with an invalid value will be treated as “Lax”. This means that the cookie will no longer be sent in third-party contexts. If your application depends on this cookie being available in such contexts, please add the “SameSite=None“ attribute to it. To know more about the “SameSite“ attribute, read https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie/SameSite [global.js:324:79](https://community.cloudflare.steamstatic.com/public/javascript/global.js?v=4vxbzv38ZWKO&l=english&_cdn=cloudflare)
Google Analytics and Tag Manager is being shimmed by Firefox. See https://bugzilla.mozilla.org/show_bug.cgi?id=1713687 for details. [sandbox eval code:1:9](chrome://devtools/content/webconsole/sandbox%20eval%20code)

```

I don't know if the `Firefox can’t establish a connection to the server at wss://community.steam-api.com/websocket/.` part is from the extension, but it's giving `403 Forbidden`

**Comment by naoliv** [comment]
There really is no reason for you to apologize.
You are very helpful and considerate, and I can only thank you.

What I see here is:
![](https://github.com/k5md/Steam-Bulk-Sell-webextension/assets/982338/50ec0688-02e3-4504-b92c-a4969808b098)


There are some different JavaScripts; for example I am seeing `economy.js` while you have `economy_v2.js`

But could it be a difference in the language?
In your requests it is using `russian`, whereas here it's using `english`.

**Comment by k5md** [comment]
>There are some different JavaScripts; for example I am seeing economy.js while you have economy_v2.js

It's because you opened devtools on steam market page (https://steamcommunity.com/market/), not on inventory (https://steamcommunity.com/my/inventory/), steam loads economy.js for market page and economy_v2.js for inventory. It's okay, I've checked both.
>But could it be a difference in the language?

You are right, I switched to english language and got the same 'v' param values and same scripts that you get and still the extension works, even with VPN enabled and on latest FF Developer Edition. Comparing ru and en yields that they differ only in localised messages inside scripts and 'v' param value is just content hash, not version.

I'm really out of ideas now, since the only reason the extension would not mount its interface and not report any errors in console is if user is not allowed to trade (I've encountered this once after inactivity), so just maybe try open console (Developer Tools -> Console tab) and type "g_bMarketAllowed" without quotes and enter. If it's falsy, then trading is blocked.

**Comment by naoliv** [comment]
> It's because you opened devtools on steam market page (https://steamcommunity.com/market/), not on inventory (https://steamcommunity.com/my/inventory/), steam loads economy.js for market page and economy_v2.js for inventory. It's okay, I've checked both.

Oh my...
I'll have to ask you a thousand apologies for this, because, apparently, I was mistakenly looking at the market page instead of the inventory.
In the inventory, it is indeed showing correctly:

![](https://github.com/k5md/Steam-Bulk-Sell-webextension/assets/982338/e5e4b2b7-84aa-441b-93b3-23a35016a591)

I truly apologize for this confusion and I'm sorry for wasting your time on it.

**Comment by k5md** [comment]
No problems! I had no idea steam guys could come to the idea of opening inventory not as a page (via redirect), but inside modal's iframe from market, such decisions are hilarious.
The issue was helpful, I'll make the extension work for this case too in a couple of days.

**Comment by k5md** [comment]
@naoliv I've released version 1.1.5 that makes the extension load in market's iframe, thanks for bringing the issue up.

**Comment by naoliv** [comment]
Great!
Thanks a lot!
